lib/views/packs/pack_list_item.dart

```diff
@@ -82,7 +83,11 @@ class PackListItemWidget extends StatelessWidget {
   Padding _coverImageWidget() {
     return Padding(
       padding: const EdgeInsets.all(8.0),
-      child: data.icon ?? NetworkImageWidget(url: data.cover!),
+      child: data.icon ??
+          NetworkImageWidget(
+            url: data.cover!,
```

Comment:
Don't use the `!` , because it will crash the app if it's null. Better to check if it's null and use a default image if it is.

Reply:
Alright let me fix this